common/Util.cpp

```diff
@@ -933,8 +933,15 @@ namespace Util
 
         char* end = nullptr;
         const size_t us = strtoul(trailing + 1, &end, 10); // Skip the '.' and read as integer.
+
+        std::size_t denominator = 1;
+        for (std::size_t i = 1; isdigit(*(trailing + i)); i++)
```

Comment:
We already know the last char* (`end`, from line 934), so only need to iterate from `trailing` to `end`. No need to call `isdigit`.

Reply:
corrected.